README.md

```diff
@@ -63,6 +63,29 @@ See [TESTING](docs/testing/README.md)
 
 There are two github workflows that runs e2e tests aginst the Kubernetes/Kubernetes Network Policy tests and the Network Policy API Working Group conformance tests.
 
+## Project Scope
+
+This project was created to fill the gap on testing coverage in the area of Network Policies.
+There are limited existing solutions, but those are specific to individual network providers that require to install additional functionality, adding complexity and debugging difficulty, and not allowing to iterate fast during the development of new features.
+
+Ref: https://github.com/kubernetes/org/issues/4856
+
+### Goals
+
+P0: Support Testing Kubernetes
+
+Stakeholders: SIG Network, SIG Testing, SIG Scalability, SIG Release, Network Policy API Working Group
+Covered Work: e2e testing, scalability, performance and reliability
+
+P0: Provide Early Feedback
+
+Stakeholders: SIG Network, Network Policy API Working Group
+Covered: New features implemented under feature gates for fast iteraton
+
+### Non Goals
+
+Implementing additional functionalities unrelated to Network Policies
+
```

Comment:
"Supporting production use" should be either a Goal or a Non-Goal. (ie, how do we handle bug reports?)

Likewise, "supporting use at scale"

Reply:
I'm trying to indicate that people is free to use it as they want and feedback is really appreciated:  "given enough eyeballs, all bugs are shallow", but this is an open source project with best effort support ...  the main goal of the project is zero bugs ... best performance ... same as we do with KIND ... don't know how to write that

Reply:
will clarify in the scope